morse.rb

```diff
@@ -0,0 +1,52 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}.freeze
+
+def decode_char(morse_char)
+  MORSE_CODE[morse_char] if MORSE_CODE.key?(morse_char)
+end
+
+def decode_word(word)
+  str = ''
+  word.split.each do |char|
+    str += decode_char(char)
+  end
+  str
+end
+
+def decode_msg(msg)
+  message = ''
+  msg.strip.tr('/', ' ').split.each do |element|
+    message += decode_word(element)
+  end
+  message
+end
```

Comment:
- [ ] Your Code is working but there is one more thing you need to fix, Your message is not adding any space to separate the words. you were supposed to return `A BOX FULL OF RUBIES` but look at your output :point_down: 

![image](https://user-images.githubusercontent.com/75126481/150144686-c17673d1-0003-475a-8a8a-1e7872de6422.png)

Kindly fix that :pray:

Reply:
Thank you , I did fix that 👍